lib/gitlab/ci/templates/pipeline/AndroidTemplate.yml

```diff
@@ -310,17 +310,78 @@ deployApkRelease:
     - if [ -z "${APK_SLACK_CHANNEL_ID}" ]; then echo "APK_SLACK_CHANNEL_ID is unset so not pushing message to Slack"; else echo "APK_SLACK_CHANNEL_ACCESS_TOKEN is set to '$APK_SLACK_CHANNEL_ID'"; fi
     - if [ -z "${DEPLOY_RELEASE_APK_SLACK_MESSAGE}" ]; then echo "$DEPLOY_RELEASE_APK_SLACK_MESSAGE is unset so will use empty message for Slack"; else echo "$DEPLOY_RELEASE_APK_SLACK_MESSAGE is set to '$DEPLOY_RELEASE_APK_SLACK_MESSAGE'"; fi
     - |
-      for DEPLOY_RELEASE_APK_NAME in $DEPLOY_RELEASE_APK_NAMES
-        do
-          release_apk_file=$(find $DEPLOY_RELEASE_APK_PATH -name *$DEPLOY_RELEASE_APK_NAME*)
-          echo "found release apk file $release_apk_file"
-          curl \
-            -F token="${APK_SLACK_CHANNEL_ACCESS_TOKEN}" \
-            -F channels="${APK_SLACK_CHANNEL_ID}" \
-            -F initial_comment="$DEPLOY_RELEASE_APK_SLACK_MESSAGE" \
-            -F "file=@$release_apk_file" \
-            https://slack.com/api/files.upload
-        done
+      for DEPLOY_DEBUG_APK_NAME in $DEPLOY_DEBUG_APK_NAMES
```

Comment:
Changing DEPLOY_RELEASE_APK_NAMES to DEPLOY_DEBUG_APK_NAMES in the `deployApkRelease` job is a breaking change. Please avoid breaking changes whether you do, or do not, add a new hidden job called `.deployApk`.